_posts/2023-03-10-Expanding-k-NN-with-Lucene-aNN.md

```diff
@@ -0,0 +1,40 @@
+---
+layout: post
+title:  "Expanding k-NN with Lucene approximate nearest neighbor search"
+authors:
+- macrakis
+date: 2023-03-10
+categories:
+ - technical-post
+meta_keywords: 
+meta_description: 
```

Comment:
**meta_keywords:** approximate k-NN, OpenSearch k-NN search, k-nearest neighbor, Lucene 9.0 k-NN

**meta_description:** Learn how approximate k-NN in OpenSearch with faiss, nmslib, and Lucene, can produce results tens of milliseconds faster than with exact K-NN